tools/nl/embeddings/input/undata_ilo/sheets_svs.csv

```diff
@@ -1,86 +1,33 @@
-dc/topic/ILOEARXTLPRT,Low Pay Rate
-dc/topic/ILOEES3EESNB,Youth Employees
-dc/topic/ILOEESXTMPRT,Share of Temporary Employees
-dc/topic/ILOEIP2EIPNB,Persons Outside The Labour Force ILO Modelled Estimates
-dc/topic/ILOEIP2JOBNB,"Potential Labour Force And Willing Non-jobseekers ILO Modelled Estimates, as Of Nov. 2023"
-dc/topic/ILOEIP2JOBRT,"Potential Labour Force And Willing Non-jobseekers Rate ILO Modelled Estimates, as Of Nov. 2023"
-dc/topic/ILOEIP2PLFRT,Potential Labour Force Rate ILO Modelled Estimates
-dc/topic/ILOEIP2WAPRT,"Number of Persons of Working Age Outside The Labour Force, as A Percentage of The Working-age Population Inactivity Rate, ILO Modelled Estimates"
-dc/topic/ILOEIP3EIPNB,Youth Outside The Labour Force
-dc/topic/ILOEIP3WAPRT,"Number of Young Persons of Working Age Outside The Labour Force, as A Percentage of The Working-age Population Youth Inactivity Rate"
+dc/topic/ILOEAR4MMNNB,"Monthly Minimum Wage"
+dc/topic/ILOEARXTLPRT,"Low Pay Rate"
+dc/topic/ILOEESXTMPRT,"Share of Temporary Employees"
 dc/topic/ILOEIP5EETNB,"Youth Not in Employment, Education or Training NEET According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
 dc/topic/ILOEIP5EETRT,"Share of Youth Not in Employment, Education or Training NEET, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
 dc/topic/ILOEIP5EIPNB,"Persons Outside The Labour Force, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
 dc/topic/ILOEIP5PLFNB,"Potential Labour Force, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
-dc/topic/ILOEMP2FTENB,Full-time Equivalent Employment
-dc/topic/ILOEMP2NIFNB,"Informal Employment ILO Modelled Estimates, as Of Nov. 2023"
-dc/topic/ILOEMP2NIFRT,"Informal Employment Rate ILO Modelled Estimates, as Of Nov. 2023"
-dc/topic/ILOEMP2TRURT,Time-related Underemployment Rate ILO Modelled Estimates
-dc/topic/ILOEMP2WAPRT,Employment-to-population Ratio ILO Modelled Estimates
-dc/topic/ILOEMP3EMPNB,Youth Employment
-dc/topic/ILOEMP3WAPRT,Youth Employment-to-population Ratio
 dc/topic/ILOEMP5NIFNB,"Informal Employment, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
 dc/topic/ILOEMP5NIFRT,"Informal Employment Rate, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
 dc/topic/ILOEMP5PIFNB,"Employment Outside The Formal Sector, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
 dc/topic/ILOEMP5PIFRT,"Share of Employment Outside The Formal Secto, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
-dc/topic/ILOEMPNIFLNB,Informal Employment
-dc/topic/ILOEMPNIFLRT,Informal Employment Rate
+dc/topic/ILOEMPNIFLRT,"Informal Employment Rate"
 dc/topic/ILOEMPNORMNB,"Employment, Normative Approach"

```

Comment:
"Employment, Normative Approach" is a bit vague?

Reply:
Added to alternatives.